schedulingApp/views.py

```diff
@@ -167,3 +167,33 @@ def get(self, request, id):
             return redirect("/")
         return render(request, "viewUser.html", {"viewing": Profile.objects.get(id=id),
                                                  "profile": Profile.objects.get(user=request.user)})
+
+class EditUser(LoginRequiredMixin, View):
+    def get(self, request, id):
+        profile = Profile.objects.get(user=request.user)
+        if profile.id != id and profile.permission != Profile.ADMIN:
+            return redirect("/")
+        return render(request, "editUser.html", {"viewing": Profile.objects.get(id=id),
+                                                 "profile": Profile.objects.get(user=request.user)})
+
```

Comment:
There doesn't appear to be anything dealing with password update in this post request.